aggregator/src/aggregator.rs

```diff
@@ -2038,22 +2097,25 @@ impl VdafOps {
 
                 Box::pin(async move {
                     // Read existing state.
-                    let (helper_aggregation_job, report_aggregations) = try_join!(
-                        tx.get_aggregation_job::<SEED_SIZE, Q, A>(task.id(), &aggregation_job_id),
-                        tx.get_report_aggregations_for_aggregation_job(
+                    let helper_aggregation_job = tx
```

Comment:
That's because at some iteration of this change, I had to add an `&A::AggregationParam` argument to `Datastore::get_report_aggregations_for_aggregation_job`. We obtain that value by calling `get_aggregation_job`, so we can't do those two things in parallel anymore.

However in some later iteration of the change, I stopped using that argument, so we can put all this back the way it was.